### ISSUE TITLE
lightwallet.js and iOS UIWebView (WkWEbView)  

### ISSUE THREAD

**Issue by 6epreu** [ConsenSys/eth-lightwallet#125]
I have tested the lightwallet.js on webveiw in IOS. 
The functions are working but executes very slow! 
Exactly the 20-30 sec timeout I have on function call 
`lightwallet.keystore.deriveKeyFromPassword(password, function(err, pwDerivedKey) {
    console.log('')
})`

to get the console.log string takes about 30 secs. 
I have tested on simulator and real device

**Comment by christianlundkvist** [comment]
Interesting, it is much faster in Mobile Safari, right?

**Comment by francesco-strazzullo** [comment]
@christianlundkvist Yes, I have the same bug. I'm using cordova and I takes minutes to create a vault and a new ethereum address. On Simulator o Safari mobile it's fast enough

**Comment by claykohut** [comment]
Anyone know how to prevent this from taking so long? I'm using react native -- it works fine on iOS but is extremely slow on android (takes around 60 seconds to create a wallet).

Thanks!
